src/components/booking/booking-card/BookingCard.vue

```diff
@@ -0,0 +1,333 @@
+<template>
+  <div class="booking-card">
+    <div :class="`wrapper ${isSmallOrDown ? 'p-5' : 'py-6 px-8'}`" @click="mobileEmit">
+
+      <div class="main-info">
+
+        <div class="head">
```

Comment:
I'm not sure about the utility of `header` here. According the doc, it's more relevant for a page header
I use the class name header because is a understandable way to know that this part is a kind of header of the card itself.
Do you think that we need that tag name anyway ?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

Reply:
Not sure. I thought this was also used for headers of parts of the page. (f.e. if you have blocks with links to other pages where you have a header and some content, I thought the header tag was also the preferable tag).

If the documentation of mozilla states otherwise, we should probably follow them.

Reply:
The article you shared clearly shows that you can use a header both for a page or for a part of a page in the examples: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header#Examples